src/App.js

```diff
@@ -54,6 +71,419 @@ const App = () => {
+              src={competenceImage}
+              width="160px"
+              height="160px"
+              alt="A greek statue"
+            />
+            <h4 className="techniques__card-title">Иллюзия компетентности</h4>
+            <p className="techniques__card-description">
+              Повторите про себя, запишите, расскажите другу: вам только
+              кажется, что вы владеете новой темой.
+            </p>
+          </li>
+        </ul>
+      </section>
+      <section className="videos">
+        <h2 className="section-title">Видео на TED</h2>
+        <h3 className="section-subtitle">
+          Для тех, кто любит прокрастинировать
+        </h3>
+        <div className="videos__youtube">
+          <iframe
+            className="videos__youtube-video"
+            width="515"
+            height="316"
+            src="https://www.youtube.com/embed/5MgBikgcWnY"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowfullscreen
+          ></iframe>
+          <iframe
+            className="videos__youtube-video"
+            width="515"
+            height="316"
+            src="https://www.youtube.com/embed/arj7oStGLkU"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowfullscreen
+          ></iframe>
+        </div>
+      </section>
+      <section className="oakley">
+        <div className="two-columns">
+          <h3 className="oakley__header">История Барбары Оакли.</h3>
+          <div className="oakley__main-text">
+            <p className="oakley__text">
+              С детства Барбаре не давалась математика. Она считала себя
+              законченным гуманитарием, причём далеко не самым умным. В армии
+              она изучала русский язык, чтобы получить надбавку, да так успешно,
+              что её выдвинули в командиры. Но для продвижения по службе нужно
+              было сдавать математику. И тогда Барбара придумала свой подход к
+              точным наукам. Оказалось, если вам что-то «плохо даётся», ваши
+              добытые трудом знания гораздо глубже, чем у тех, кому всё ясно с
+              первого взгляда.
+            </p>
+          </div>
+        </div>
+      </section>
+      <section className="feynman" height="890px">
+        <h1 className="feynman__title">Метод Фейнмана</h1>
+        <h2 className="feynman__subtitle">Выучить и не забыть.</h2>
+        <a
+          href="https://en.wikipedia.org/wiki/Wiki"
+          className="feynman__link  transparent-link"
+        >
+          Подробнее →
+        </a>
+      </section>
+
+      <section className="digits">
+        <h2 className="section-title">Цифры и факты</h2>
+        <h3 className="section-subtitle">Про учебу и мозг</h3>
```

Comment:
```suggestion
        <h3 className="digits__section-subtitle">Про учебу и мозг</h3>
```
fix the CSS selectors according to the change

Reply:
outdated